packages/components/src/dimension-control/test/index.test.js

```diff
@@ -32,7 +37,7 @@ describe( 'DimensionControl', () => {
 				<DimensionControl
 					instanceId={ uniqueId() }
 					label={ 'Margin' }
-					icon={ 'tablet' }
+					icon={ plus }
```

Comment:
This change isn't related to the icon change, but is a matter of using a different icon for the dimension control, correct? If so, all good. We'll want a "tablet" icon at some point.

Reply:
This is just a test, I didn't want to create an icon for something that is used in a test. We may need a tablet icon later but we don't need it today anywhere in the UI.